lib/stateful_model_rails/state_machine.rb

```diff
@@ -133,11 +141,11 @@ def self.included(base)
   end
 end
 
-def included__state_machine(opts = {}, &block)
+def included__state_machine(opts = {}, &)
```

Comment:
It's weird to me that this takes `opts` then just extracts `on`. Not a change here but it's striking to look at.

Reply:
#futureproof